src/lib/components/Pagination/PaginationButton.tsx

```diff
@@ -23,6 +25,7 @@ const PaginationButton: FC<PaginationButtonProps> = ({
   className,
   onClick,
   theme: customTheme = {},
+  disabled,
```

Comment:
`disabled` isn't the same as that `active = false`?

Reply:
Or does it have another meaning here? Can it be `active` and `disabled` at the same time?

Reply:
checking the code, it seems to be possible... but my question is more about the logic itself. :)

Reply:
@rluders  Here I want to use `disabled` state only on the Prev and Next buttons and not the other pagination buttons. Prev and Next buttons will never have `active` state. 

But since all the pagination buttons including Prev and Next use PaginationButton component, I added `disabled` state there. I understand your point that `disabled` state wont make sense for PaginationButton component.

I guess I can create a separate Button to use for Next and Previous buttons so to avoid adding `disabled` state on the PaginationButton component.

What do you suggest ?

Reply:
@rluders I have updated the code with new changes. Please take look and let me know your opinion.